packages/kbn-index-adapter/src/field_maps/types.ts

```diff
@@ -46,6 +46,7 @@ export type FieldMap<T extends string = string> = Record<
     array?: boolean;
     doc_values?: boolean;
     enabled?: boolean;
+    fields?: Record<string, { type: string }>;
```

Comment:
@semd I extended FieldMap to have `fields` as well. We need it to be able to do sort by `title` field which has `text` type. And in order to sort by this field we need to add sub-field of a `keyword` type.

https://www.elastic.co/guide/en/elasticsearch/reference/current/text.html#before-enabling-fielddata